docs/3.0.0-beta.x/getting-started/deployment.md

```diff
@@ -0,0 +1,190 @@
+# Deployment
+
+Strapi gives you many possible deployment options for your project or application. Strapi can be deployed on traditional hosting servers or services such as Heroku, AWS, Azure and others. The following documentation covers how to develop locally with Strapi and deploy Strapi with various hosting options.
+
+::: tip
+Deploying **databases** along with Strapi is covered in the [Databases Guide](../guides/databases.md).
+:::
+
+### Documented Options
```

Comment:
I think we can rename this section with a `##` and named it something like `Hosting provider guides`
Just to mention that the followings are deployment guides for a specific platform.